tests/e2e/tests/AggregatesSection/AggregatesShouldNotBeZeroInTriggerState.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@playwright/test';
+import AggregatesComponent from 'Pages/AggregatesComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { TriggerDataSet } from 'testData/testData.enum';
+
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    TriggerDataSet.TriggerScenario,
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(39, '[Trigger] Aggregates values should be non-zero'),
```

Comment:
You started at some point to add qase links above the qase id here, which I found pretty useful. Do you agree it's relevant to keep doing that? If not too much trouble, maybe add it for all tests so far?

Reply:
Ohh yes I can still do that. I will update it.

Reply:
Is it easy to maybe add it now to all tests so far?

Reply:
I've added them